osp/osp.c

```diff
@@ -152,7 +152,8 @@ osp_connection_new (const char *host, int port, const char *cacert,
         }
 
       addr.sun_family = AF_UNIX;
-      strncpy (addr.sun_path, host, sizeof (addr.sun_path) - 1);
+      memset (addr.sun_path, 0, sizeof (addr.sun_path) - 1);
```

Comment:
```suggestion
      memset (addr.sun_path, 0, sizeof (addr.sun_path));
```

I guess we want to initialize everything?